torchmetrics/functional/audio/snr.py

```diff
@@ -64,3 +65,31 @@ def snr(preds: Tensor, target: Tensor, zero_mean: bool = False) -> Tensor:
     snr_value = 10 * torch.log10(snr_value)
 
     return snr_value
+
+
+def sisnr(preds: Tensor, target: Tensor) -> Tensor:
+    """Scale-invariant signal-to-noise ratio (SI-SNR).
+
+    Args:
+        preds:
+            shape ``[...,time]``
+        target:
+            shape ``[...,time]``
+
+    Returns:
+        si-snr value of shape [...]
+
+    Example:
+        >>> import torch
+        >>> from torchmetrics.functional.audio import sisnr
+        >>> target = torch.tensor([3.0, -0.5, 2.0, 7.0])
+        >>> preds = torch.tensor([2.5, 0.0, 2.0, 8.0])
+        >>> sisnr(preds, target)
+        tensor(15.0918)
+
+    References:
+        [1] Y. Luo and N. Mesgarani, "TaSNet: Time-Domain Audio Separation Network for Real-Time, Single-Channel Speech
+        Separation," 2018 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP), 2018, pp.
+        696-700, doi: 10.1109/ICASSP.2018.8462116.
+    """
+    return si_sdr(target=target, preds=preds, zero_mean=True)
```

Comment:
maybe we can consider move this to a new file `sisdr.py`, which has `sisdr` and `sisnr`

Reply:
yes, after the other one is merged